common/src/main/java/io/netty/util/AsciiString.java

```diff
@@ -1042,4 +833,131 @@ public boolean contains(CharSequence cs) {
         }
         return indexOf(cs) >= 0;
     }
+
+    private static byte toLowerCase(byte b) {
+        if ('A' <= b && b <= 'Z') {
+            return (byte) (b + UPPER_CASE_TO_LOWER_CASE_ASCII_OFFSET);
```

Comment:
can we just inline the value of `UPPER_CASE_TO_LOWER_CASE_ASCII_OFFSET`. This is just a meaningless constant and it won't ever change.